Build.bee.cs

```diff
@@ -249,7 +249,8 @@ static NPath GetBuildTargetDir(ToolChain toolchain, bool isExecutable = false)
             {"mac_x64", "macos/x64"},
             {"mac_arm64", "macos/arm64"},
             {"linux_x64_clang", "linux"},
-            {"win64_vs2017", "win64"},
+            {"win_x64_vs2017_win7", "win/x64"},
```

Comment:
Do we really build x64 and arm64 with two different VS versions? Weird.

Reply:
Yes -- afaik there's no real reason why we do this, but I left it as it was in the name of not making unnecessary changes :-)